java/client/src/test/java/glide/api/RedisClientTest.java

```diff
@@ -3955,75 +3955,76 @@ private static List<Arguments> getStreamAddOptions() {
-                                new String[] {
-                                    TRIM_MAXLEN_REDIS_API,
-                                    TRIM_EXACT_REDIS_API,
-                                    Long.toString(5L),
-                                    TRIM_LIMIT_REDIS_API,
-                                    Long.toString(10L),
-                                    "id"
-                                }),
-                        Arguments.of(
-                                // MAXLEN with non exact match
-                                "test_xadd_maxlen_with_non_exact_match",
-                                StreamAddOptions.builder()
-                                        .makeStream(Boolean.FALSE)
-                                        .trim(new MaxLen(false, 2L))
-                                        .build(),
-                                new String[] {
-                                    NO_MAKE_STREAM_REDIS_API,
-                                    TRIM_MAXLEN_REDIS_API,
-                                    TRIM_NOT_EXACT_REDIS_API,
-                                    Long.toString(2L),
-                                    "*"
-                                }),
-                        Arguments.of(
-                                // MIN ID with LIMIT
-                                "test_xadd_minid_with_limit",
-                                StreamAddOptions.builder()
-                                        .id("id")
-                                        .makeStream(Boolean.TRUE)
-                                        .trim(new MinId("testKey", 10L))
-                                        .build(),
-                                new String[] {
-                                    TRIM_MINID_REDIS_API,
-                                    TRIM_EXACT_REDIS_API,
-                                    Long.toString(5L),
-                                    TRIM_LIMIT_REDIS_API,
-                                    Long.toString(10L),
-                                    "id"
-                                }),
-                        Arguments.of(
-                                // MIN ID with non exact match
-                                "test_xadd_minid_with_non_exact_match",
-                                StreamAddOptions.builder()
-                                        .makeStream(Boolean.FALSE)
-                                        .trim(new MinId(false, "testKey"))
-                                        .build(),
-                                new String[] {
-                                    NO_MAKE_STREAM_REDIS_API,
-                                    TRIM_MINID_REDIS_API,
-                                    TRIM_NOT_EXACT_REDIS_API,
-                                    Long.toString(5L),
-                                    "*"
-                                })));
+                        new String[] {NO_MAKE_STREAM_REDIS_API, "id"}),
```

Comment:
THIS bracket was missing... causing only the first test to run

Reply:
... And made spotless crazy once you add it

Reply:
that was the first sign...